lib/eslint6-patch.js

```diff
@@ -1,7 +1,11 @@
 "use strict"
 
-const { ConfigArrayFactory } = require("eslint/lib/cli-engine/config-array-factory")
-const { ConfigArray } = require("eslint/lib/cli-engine/config-array")
+const {
+  Legacy: {
+      ConfigArray,
+      ConfigArrayFactory
+  }
+} = require("@eslint/eslintrc")
```

Comment:
it may be worth it to invest some time on figuring how to use the new versions of ConfigArrayFactory and ConfigArray ? Also, what do you you think about renaming this file to `eslint8-patch.js` instead ?

Reply:
okay, that makes sense